drivers/storage/portworx/deployment.go

```diff
@@ -816,6 +816,15 @@ func (t *template) getEnvList() []v1.EnvVar {
 			Name:  pxutil.EnvKeyPortworxSecretsNamespace,
 			Value: t.cluster.Namespace,
 		},
+		"NODE_NAME": {
+			Name: "NODE_NAME",
+			ValueFrom: &v1.EnvVarSource{
+				FieldRef: &v1.ObjectFieldSelector{
+					APIVersion: "v1",
+					FieldPath:  "spec.nodeName",
```

Comment:
Do you need the quotes "" around spec.nodeName ?

Reply:
Yes, because this is a field reference. Kubernetes populates the env value in the pod using this field reference. The nodename is not taken from the code here.